doc/index.rst

```diff
@@ -161,8 +161,9 @@ The main option is ``root_sass`` option, which defaults to ``assets/styles/app.s
 .. code-block:: yaml
 
     # config/packages/symfonycasts_sass.yaml
-    symfonycasts_sass:
-        root_sass:  'assets/styles/app.scss'
+    root_sass:
```

Comment:
Yeah @smnandre has a good point, we should bring `symfonycasts_sass` root key as well, so it should be:

```yaml
symfonycasts_sass:
    root_sass:
        # Default:
        - %kernel.project_dir%/assets/styles/app.scss
```

Could you apply this change?

Reply:
sure, thanks @smnandre @bocharsky-bw